src/coreclr/jit/instr.h

```diff
@@ -376,6 +376,8 @@ enum insScalableOpts : unsigned
 
     INS_SCALABLE_OPTS_WITH_VECTOR_PAIR,    // Variants with {<Zn1>.<T>, <Zn2>.<T>} sve register pair (eg splice)
 
+    INS_SCALABLE_OPTS_IMM_BITMASK,         // Variants with an immediate that is a bitmask
```

Comment:
This may not be the cleanest interface, but if we want to avoid creating new insScalableOpts entries, we could have the caller to `emitIns_*` always use `INS_sve_dupm`, and then rely on `emitIns` to use `useMovDisasmForBitMask` to decide which disasm should be used. But that might be confusing from the caller's perspective if we cannot pass `INS_sve_mov`.